src/main/java/org/traccar/model/Position.java

```diff
@@ -44,6 +44,9 @@ public class Position extends Message {
     public static final String KEY_ODOMETER_TRIP = "tripOdometer"; // meters
     public static final String KEY_HOURS = "hours"; // milliseconds
     public static final String KEY_STEPS = "steps";
+    public static final String KEY_BLOOD_OXYGEN = "bloodOxygen";
```

Comment:
We only add new parameters here where they are used by at least 2 protocols. You would also have to update the web app with those new attributes.

Reply:
I think the web app can handle this update automatically.

![image](https://github.com/traccar/traccar/assets/26859021/49c2886e-6764-4436-a416-c800a3ca83ba)

![image](https://github.com/traccar/traccar/assets/26859021/8fa7b390-8bf8-4aaf-acb2-808f454e5236)

Do i have to change something else in the web app ?

Reply:
No, it cannot. Known attributes have to be proper translatable attributes.